test/testdata/resolver/alias_transitive.rb

```diff
@@ -0,0 +1,30 @@
+# typed: true
+
+class Parent
+  extend T::Sig
+
+  sig {returns(Integer)}
+  def parent_bar; 1; end
+end
+
+class Child < Parent
+  alias_method(:child_bar, :parent_bar)
+end
+
+T.reveal_type(Child.new.child_bar) # Revealed type: `Integer`
+
+# --- again, but in a different order ---
+
+class ChildFirst < ParentSecond
+  alias_method(:foo, :bar)
+end
+
+class ParentSecond
+  extend T::Sig
+
+  sig {returns(NilClass)}
+  def bar
+  end
+end
```

Comment:
Given that the second case doesn't work at runtime, I think the worries I had are unfounded.

Reply:
Note that on second run in IDE, we'll loose the prior knowledge from the order